WordPress/src/main/java/org/wordpress/android/ui/prefs/SiteSettingsFragment.java

```diff
@@ -692,20 +689,67 @@ public String format(int value) {
         });
     }
 
-    private void showExportContentDialog() {
+    private void requestPurchasesForDeletionCheck() {
+        final Blog currentBlog = WordPress.getCurrentBlog();
+        WordPress.getRestClientUtils().getSitePurchases(currentBlog.getDotComBlogId(), new RestRequest.Listener() {
+            @Override
+            public void onResponse(JSONObject response) {
+                showPurchasesOrDeleteSiteDialog(response, currentBlog);
+            }
+        }, new RestRequest.ErrorListener() {
+            @Override
+            public void onErrorResponse(VolleyError error) {
+                ToastUtils.showToast(getActivity(), getString(R.string.purchases_request_error));
+                AppLog.e(AppLog.T.API, error.toString());
```

Comment:
Maybe add a description for the error here, something like `"Error occurred while requesting purchases for deletion check: " + error.toString()`. I think it'd be easier for us to find the issue from the logs, especially for the HEs.